frontend/src/component/feature/FeatureToggleList/FeatureToggleListTable.tsx

```diff
@@ -288,14 +285,7 @@ const FeatureToggleListTableComponent: VFC = () => {
                                     </>
                                 }
                             />
-                            <Link
```

Comment:
why is archive removed in this PR?

Reply:
This was the decision from yesterday, to remove archive.